pkg/cache/resource.go

```diff
@@ -30,10 +30,51 @@ type Resource interface {
 
 // Common names for Envoy filters.
 const (
-	CORS                  = "envoy.cors"
-	Router                = "envoy.router"
+	// Buffer HTTP filter
+	Buffer = "envoy.buffer"
```

Comment:
is `cache` the right package for these?

Reply:
It's not ideal. We have "cache", "server", "util", "test", "log" packages. Which one would be better? Should we create a new one "filter", maybe?

Reply:
cc @ostromart since he recently touched this code

Reply:
I would say either util or filter.

Reply:
In the interest of package economy, I'll move these well-known names to util.